src/util/helpers.js

```diff
@@ -6,6 +6,17 @@ const mkdirp = require("mkdirp")
 
 const log = require("./log")
 
+/**
+ * Adds the number of unread messages in the terminal title
+ * @param {Object} unreadMessagesCount number of unread messages
+ */
+function notifyTerminal(unreadMessagesCount) {
+  const title = (unreadMessagesCount) ? `messer (${unreadMessagesCount})` : "messer";
+  process.stdout.write(
+    String.fromCharCode(27) + "]0;" + title + String.fromCharCode(7)
```

Comment:
Pretty sure this will cause the linter to fail - I think it wants template literal i.e.
```js
`${String.fromCharCode(27)}]0;${title}${String.fromCharCode(7)}`
```